HW02/Kate_Fedotova/kate_fedotova_hw_02_t_02.rb

```diff
@@ -0,0 +1,24 @@
+if File.open('task_2.txt')
+  puts 'File open:' + 'task_2.txt'
+end
+
+unless File.exist?('task_2.txt')
+  abort 'File is not exist!'
+end
+
+logs = File.readlines('task_2.txt')
+  
+def new_format(lines)
+  array = []
+  lines.each do |line|
+    if line.match(/^\d{1,3}\.\d{1,3}\.\d*\.\d* \- \- \[\d.\/+\w*\/\d*\:\d.\:\d.\:\d. \+\d*\] \"\w* \/\w*\/\d\/\w* \w*\/\d\.\d\"/)
+      ip = line.match(/\d{1,3}\.\d{1,3}\.\d*\.\d*/)
```

Comment:
We can move the same regular expressions to constant 
```suggestion
  IP = '\d{1,3}\.\d{1,3}\.\d*\.\d*'
  lines.each do |line|
    if line.match(/^#{IP} \- \- \[\d.\/+\w*\/\d*\:\d.\:\d.\:\d. \+\d*\] \"\w* \/\w*\/\d\/\w* \w*\/\d\.\d\"/)
      ip = line.match(/#{IP}/)
```